app/config/config.yml

```diff
@@ -55,8 +55,8 @@ framework:
     strict_requirements: ~
   form: ~
   csrf_protection: ~
-  validation: { enable_annotations: true }
-  serializer: { enable_annotations: true }
+  validation: { enable_attributes: true }
+  serializer: { enable_attributes: true }
```

Comment:
It is not possible to have enable_annotations, it is also not possible to have both values.